documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,434 @@
+
+.Removed Cancel icon during Cutover stage
+
+In a warm migration, the Cancel icon appears during the Precopy stage, but does not appear during the Cutover stage. link:https://issues.redhat.com/browse/MTV-449[(MTV-449)]
+
+.{project-short} UI plugin uses only a dynamic plugin for SDK REST API fetch methods
+
+The UI plugin uses a dynamic plugin for SDK REST API when using the `fetch` method. link:https://issues.redhat.com/browse/MTV-616[(MTV-616)]
+
+.Improved cell renderers in list of provider VMs
+
+The table listing providers and their VMs has been remapped and improved to show the relevant fields for each provider. link:https://issues.redhat.com/browse/MTV-773[(MTV-773)]
+
+.Date filter
+
+Lists of CRS can be filtered by different criteria, including creation date. link:https://issues.redhat.com/browse/MTV-724[(MTV-724)]
+
+.Improved explanations in providers form
+
+The `providers` form runs a validation to verify the text entered in each field is correct and relevant for that field. A `hot to fix` explanation appears when incorrect or badly formatted text is entered in each field. link:https://issues.redhat.com/browse/MTV-766[(MTV-766)]
+
+.Create provider form includes information hint
+
+Fields in the `create provider` form include hint icons to explain the required information for each field. link:https://issues.redhat.com/browse/MTV-768[(MTV-768)]
+
+.Migration success status bar chart 
+
+A bar chart shows the number of migrations and their status - `startes`, `finished`, and `failed`. link:https://issues.redhat.com/browse/MTV-770[(MTV-770)]
+
+.Improved list of VMs in the detailed view of each provider
+
+The detailed view of each provider includes specific information relevant to the VMs of that provider. link:https://issues.redhat.com/browse/MTV-772[(MTV-772)]
+
+.Automated kubevit types
+
+The process of generating `kubevit` types has been automated. link:https://issues.redhat.com/browse/MTV-775[(MTV-775)]
+
+.Edit plans that failed to migrate VMs
+
+Plans that have failed to migrate any VMs can be edited. Some plans fail or are canceled because the network and storage mappings are wrong. These plans can be edited until they succeed. link:https://issues.redhat.com/browse/MTV-779[(MTV-779)]
+
+.Specify ESX provider
+
+An ESX can be imported directly from vSphere without going through vCenter. In the provider screen, the user can enter a URL that points to vCenter or to a specific ESX. link:https://issues.redhat.com/browse/MTV-792[(MTV-792)]
+
+.ESXi provider host configuration
+
+ESXi as a vSphere provider does not need credentials in its setup. The credentials requirements field is removed from ESX settings. link:https://issues.redhat.com/browse/MTV-793[(MTV-793)]
+
+.Link to VM details page
+
+The {ocp-name} Console, provider VM list includes a link to the details page for each VM. link:https://issues.redhat.com/browse/MTV-797[(MTV-797)]
```

Comment:
Is it the details page or the Details page?

Reply:
@ahadas 
Does the provider VM appear in the UI?
What does the page or look look like?